webapp/js/components/Utils.js

```diff
@@ -23,3 +23,35 @@ export function T(message) {
     return msg
 }
 
+export function isHTTPError(response) {
+    // Check for an HTTP error
+    if ((response.statusCode >= 300) && (response.statusCode != 400)) {
```

Comment:
Not sure that 3xx codes can be considered errors, but 400 is indeed an error. What's the purpose of this method? Maybe the name or the conditions to return true/false should be changed?